changelogs/unreleased/gh-5444-relax-uuid-validation.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/core
+
+* Any 128-byte sequence is considered as a valid UUID value (gh-5444).
```

Comment:
The issue is marked with the `feature` label. I don't mind classifying it as a feature or as a bugfix, but I'm for consistency. If it is a bugfix, then the issue should be marked accordingly and the fix should be backported down to 2.11. If it is a feature, the changelog entry should be adjusted and the patch should go to 3.2 only.

Disregarding to the feature/bugfix question, I would add a documentation request that asks to make it clear in the uuid module documentation how different tarantool versions behaves in context of UUID parsing/processing.

Reply:
Replaced with a `feature` label in changelogs. Added the corresponding TarantolBot document for the documentation request.